Doc/using/cmdline.rst

```diff
@@ -583,6 +599,14 @@ conflict.
    within a Python program as the variable :data:`sys.path`.
 
 
+.. envvar:: PYTHONDONTADDPATH0
```

Comment:
You're emphasizing `sys.path[0]` again. There's always a `sys.path[0]`. How about `PYTHONUSESAFEPATH`?

Reply:
How does the doc call the thing that is prepended?  Main script base directory?  Could be  `PYTHONDONTADDMAINDIR` or `PYTHONNOMAINDIR`.